rustls/src/client/handy.rs

```diff
@@ -5,10 +5,10 @@ use crate::msgs::persist;
 use crate::sign;
 use crate::NamedGroup;
 use crate::ServerName;
-#[cfg(feature = "ring")]
-use crate::{crypto::ring, error::Error};
+#[cfg(any(feature = "ring", feature = "aws_lc_rs"))]
```

Comment:
I think this commit should probably be squashed into one of the earlier commits.